javaslang/src-gen/main/java/javaslang/API.java

```diff
@@ -84,6 +84,280 @@ private API() {
     }
 
     //
+    // Aliases for static factories
```

Comment:
Right, but it's not my decision - I am only follow current code style: 
* https://github.com/javaslang/javaslang/blob/master/javaslang/src-gen/main/java/javaslang/API.java#L87
* https://github.com/javaslang/javaslang/blob/master/javaslang/src-gen/main/java/javaslang/API.java#L109
* https://github.com/javaslang/javaslang/blob/master/javaslang/src-gen/main/java/javaslang/API.java#L598